asdf/tests/data/custom_schema_definitions.yaml

```diff
@@ -1,9 +1,7 @@
 %YAML 1.1
 ---
 $schema: "http://stsci.edu/schemas/yaml-schema/draft-01"
-id: "http://stsci.edu/schemas/asdf/core/asdf-1.1.0"
```

Comment:
Re-using the core schema id causes problems when resolving external references.  I don't think it's a good idea anyway, since the ids are meant to be globally unique.